CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-# dbt_mailchimp_source version.version
+# dbt_mailchimp_source v0.7.0
+This release includes the following updates.
+
+## Breaking Changes (`--full-refresh` required)
+- Fixed materialization configuration in `dbt_project.yml` to ensure tmp models are materialized as views rather than tables. ([#25](https://github.com/fivetran/dbt_mailchimp_source/pull/25)) 
```

Comment:
This is a schema change as well right? Should we have the schema change matrix in here then